scss/_base_grid-definitions.scss

```diff
@@ -112,6 +112,48 @@
     }
   }
 
+  %vf-grid-row-equal-height {
+    $horizontal-list-section-medium-content-width: 66ch;
+    $horizontal-list-section-large-content-width: 100ch;
+    @extend %fixed-width-container;
```

Comment:
This mixes responsive styling with intrinsic styling, I'd suggest avoiding it if possible
https://github.com/canonical/vanilla-framework/blob/21fb5bbc409764a6c39b53a4de2a00eb489f15c8/scss/_base_grid-definitions.scss#L115

Maybe we should have a mixin for intrinsically-styled grids so that we can reusably create styles from fixed-width-container, that way components don't need to reinvent the wheel just to get intrinsic grids working.